packages/twenty-server/src/engine/metadata-modules/agent/agent-tool.service.ts

```diff
@@ -7,11 +7,13 @@ import { z } from 'zod';
 
 import { ToolAdapterService } from 'src/engine/core-modules/ai/services/tool-adapter.service';
```

Comment:
For the Metadata MCP Antoine had created services like CreateToolsService, etc.
Let's move this into a dedicated tools sections and start to consolidate things, with something clearly dedicated to workflow tools